### ISSUE TITLE
Version 1.1.1.1 is considered as valid version 

### ISSUE THREAD

**Issue by obfischer** [vdurmont/semver4j#60]
Version 3.10.0 considers the version `1.1.1.1` as valid semantic version number. According to the semantic versioning specification, the version core must consist ony of the major version, the minor version and the patch version. Please accept only valid semantic version numbers.

**Comment by piotrooo** [comment]
@obfischer if you are still interested, I've create active copy of the semver4j lib (https://github.com/semver4j/semver4j) and fix this issue in 1.0.1 version.

**Comment by obfischer** [comment]
@piotrooo Yes, of course. I also thought about forking it. I will switch to your version.

**Comment by piotrooo** [comment]
@obfischer I have in plans to do a big refactor since 2.0.0.
I have in plans to address all (or most) issues from this lib
